start.sh

```diff
@@ -38,34 +49,35 @@ EOI
 ┃ Removing previous saved environment...                                                      |
 ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
 EOI
-    rm .env-saved
+    rm ${ENV_FILE}
     fi
 fi
 
-if [ ! -f .env-saved ]
+if [ ! -f ${ENV_FILE} ]
 then
     # Setup some variables for easy port management
-    STARTING_PORT=3010
+    read -p "Enter starting port for local port mapping [3010]: " portno
+    STARTING_PORT=${portno:-3010}
     for i in {0..10}
     do
     eval SERVICE_PORT_${i}=$(( STARTING_PORT + i ))
     eval "export SERVICE_PORT_${i}=\${SERVICE_PORT_${i}}"
-    eval "echo SERVICE_PORT_${i}=\${SERVICE_PORT_${i}}" >> .env-saved
+    eval "echo SERVICE_PORT_${i}=\${SERVICE_PORT_${i}}" >> ${ENV_FILE}
     done
 
-    # Create .env-saved file to store environment variables
+    # Create ${ENV_FILE} file to store environment variables
     cat << EOI
 ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ Creating .env-saved file to store environment variables...                                  ┃
+┃ Creating ${ENV_FILE} file to store environment variables...                                  ┃
```

Comment:
nit: remove pipes to eliminate format problems with dynamic names
```suggestion
  Creating ${ENV_FILE} file to store environment variables...
```